js/contraster.js

```diff
@@ -3,7 +3,8 @@ namespacer('bcpl');
 bcpl.contraster = (($, browserStorage) => {
 	const contrasterDefaults = {
 		styleSheet: {
-			high: '/sebin/x/v/master-high-contrast.min.css'
+			high: '/sebin/x/v/master-high-contrast.min.css',
```

Comment:
i actually think of adding all this extra logic for polaris we could make the high style sheet absolute.

`//bcpl.info/sebin/x/v/master-high-contrast.min.css`

Reply:
i'd love to not check the url is possible, but my guess is probably the high-contrast-stylesheet is different, so this can stay as is, right now.